shared/attributes.asciidoc

```diff
@@ -155,9 +159,9 @@
 :cdataframes:                continuous {dataframes}
 :cdataframe-cap:             Continuous {dataframe}
 :cdataframes-cap:            Continuous {dataframes}
-:cdataframe-transform:       continuous {dataframe-transform}
-:cdataframe-transforms:      continuous {dataframe-transforms}
-:cdataframe-transforms-cap:  Continuous {dataframe-transforms}
+:cdataframe-transform:       continuous {transform}
```

Comment:
Should we create "ctransform" attributes (or something like that) too?

Reply:
Yes, I'll add them.